core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/Tenant.java

```diff
@@ -32,8 +35,10 @@
 
     private String id;
 
+    @JsonInclude(Include.NON_EMPTY)
     private List<AggregationTemplate> aggregationTemplates = new ArrayList<>();
```

Comment:
On the up side, it make our lives easier to annotate the model in the core-api library. On the flip side, now you need to get a Jackson dependency if you want to embed the Metrics core library.